.ci/magician/cmd/generate_comment.go

```diff
@@ -71,6 +69,13 @@ var generateCommentCmd = &cobra.Command{
 			env[ev] = val
 		}
 
+		for _, tokenName := range []string{"GITHUB_TOKEN_DOWNSTREAMS", "GITHUB_TOKEN_MAGIC_MODULES"} {
+			val, ok := lookupGithubTokenOrFallback(tokenName)
+			if ok {
+				env[tokenName] = val
+				break
```

Comment:
I was wrong - this shouldn't have a break - both need to get set if present.

```suggestion
```

Reply:
actually, I may have been wrong altogether and the previous logic was correct, since we _do_ want to require both of these to be set.